backend/document-service/src/main/kotlin/com/nosql/document/dto/PersonDto.kt

```diff
@@ -0,0 +1,15 @@
+package com.nosql.document.dto
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+class PersonDto(
```

Comment:
use `data class`

Reply:
this make no sense, we do not use toString